ui/widgets/datepicker.js

```diff
@@ -863,6 +865,10 @@ $.extend( Datepicker.prototype, {
 				origyearshtml = inst.yearshtml = null;
 			}, 0 );
 		}
+
+		if ( onUpdateDatepicker ) {
+			onUpdateDatepicker.apply( ( inst.input ? inst.input[ 0 ] : null ), [ inst ] );
```

Comment:
When can `inst.input` be not defined at this point?

Reply:
@mgol I was just copying the existing pattern for callbacks in this widget for consistency:

https://github.com/jquery/jquery-ui/blob/b864cd103a0acb76b0a34fb1dd382dc0925ef9a8/ui/widgets/datepicker.js#L647-L648

I'm happy to change it if you think `.call()` is better.

I don't know if `inst.inpu` is always defined or not. Again, I was copying the existing pattern here and figured it was safer to code it to handle it being `null` just in case, rather than introduce a bug. Happy to change it if I should.

Reply:
Oh, I see that this is following the existing patterns now. It's better to leave it as-is then, thanks!